etc/config-template.toml

```diff
@@ -64,6 +64,9 @@
 ## If not set, `addr` will be used.
 # advertise-addr = ""
 
+## HTTP listening address.
+# http-addr = "127.0.0.1:0". If it is not specified, the system will use a random port.
```

Comment:
here seem we can't use a default port for backward compatibility. Let's consider the following case, the user has already deployed multi TiKV instances in one machine with ansible, then he/she only just upgrades binary but forgets to add the HTTP port, so all the TiKVs in one machine use one port, and I guess only the first TiKV can start successfully.

Reply:
A random port isn't guaranteed to not have a conflict. Additionally, the behavior will be that it doesn't work anyways (the metrics won't be found). My suggestion would be
* use a standard port
* if there is a port conflict
  * don't serve metrics
  * log an error and startup

Reply:
yes, if port conflicts, we can log an error and don't serve HTTP.